clang/lib/Sema/SemaDecl.cpp

```diff
@@ -1297,6 +1297,14 @@ ExprResult Sema::ActOnNameClassifiedAsOverloadSet(Scope *S, Expr *E) {
   return ULE;
 }
 
+// ExprResult Sema::ActOnnameofOperator(Scope *S,
+//                                               SourceLocation OpLoc,
+//                                               IdentifierInfo &Name,
+//                                               SourceLocation NameLoc,
+//                                               SourceLocation RParenLoc){
+
+//                                               }
+
```

Comment:
I have already mentioned about these changes in the previous review. Please remove all the unnecessary changes such as whitespaces and unnecessary comments to keep the code clean